package.json

```diff
@@ -55,8 +55,9 @@
     "test:platform:saucelabs": "node --max_old_space_size=4096 ./node_modules/.bin/nx test platform --watch false --code-coverage true --karmaConfig=karma.conf-ci.js",
     "active-test": "nx test core --watch true --browsers=ChromeHeadless",
     "test:coveralls": "cat ./coverage/libs/core/lcov.info  | ./node_modules/coveralls/bin/coveralls.js || echo -e \"Coveralls failed.\"",
-    "compile-typedoc-all": "npm run compile-typedoc-core && npm run compile-typedoc-platform",
+    "compile-typedoc-all": "npm run compile-typedoc-core && npm run compile-typedoc-platform && npm run compile-typedoc-moment-adapter",
     "compile-typedoc-core": "rm -rf apps/docs/src/assets/typedoc/core && typedoc --out apps/docs/src/assets/typedoc/core libs/core/src/lib --tsconfig libs/core/src/lib/tsconfig.lib.json --hideGenerator --theme apps/docs/src/fd-typedoc",
+    "compile-typedoc-moment-adapter": "rm -rf apps/docs/src/assets/typedoc/moment-adapter && typedoc --out apps/docs/src/assets/typedoc/moment-adapter libs/moment-adapter/src/lib --tsconfig libs/moment-adapter/tsconfig.lib.json --hideGenerator --theme apps/docs/src/fd-typedoc",
```

Comment:
I can't see publish command for the moment adapter library.

Reply:
updated ci-scripts